samples/integration/vcpkg/src/main.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+/**
+ * @brief This sample provides the code implementation to use the Key Vault SDK client for C++
+ * to create, get, update, delete and purge a key.
+ *
+ * @remark The following environment variables must be set before running the sample.
+ * - AZURE_KEYVAULT_URL:  To the KeyVault account url.
+ * - AZURE_TENANT_ID:     Tenant id for the Azure account.
+ * - AZURE_CLIENT_ID:     The client id to authenticate the request.
+ * - AZURE_CLIENT_SECRET: The secret id from the client id.
+ *
+ */
+#include <azure/core.hpp>
+#include <azure/identity.hpp>
+#include <azure/keyvault/key_vault.hpp>
+
+#include <chrono>
+#include <iostream>
+#include <memory>
+#include <thread>
+
+using namespace Azure::Security::KeyVault::Keys;
```

Comment:
Since KeyVault isn't GA'ing yet, let's use a storage upload/download blob sample instead.

Reply:
Also, the sample seems quite busy, can we simplify it a bit? We want this as a quick starting point for customers to copy/paste and get started. We don't need it to do too much.